src/game.cpp

```diff
@@ -7685,8 +7685,12 @@ void game::examine( const tripoint &examp )
                 return;
             }
         }
+        if( mon != nullptr && mon->has_flag( MF_MILKABLE ) ) {
+            add_msg( m_info, _( "This is a milkable cow." ) );
```

Comment:
Right, my bad. I forgot to put the thing that reads whatever animal you are looking.